<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homescreen</class>
 <widget class="QWidget" name="homescreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="batteryBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(87, 227, 137);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Battery</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="insulinBar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="insulinLevelLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Insulin Levels</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QLabel" name="settingsLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>INSULIN ON BOARD (IOB)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="iobDecorButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>285</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="iobDectorButton2">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>285</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="iobUnitsLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>295</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Units</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobTimeLabel">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>295</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Remaining</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobUnitsUnit">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>295</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobUnits2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>295</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hrs</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobUnits">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>295</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobTime">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>295</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bolusButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>315</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 25pt &quot;DejaVu Sans&quot;;
text-align: left;
</string>
   </property>
   <property name="text">
    <string>BOLUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="optionsButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>315</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 25pt &quot;DejaVu Sans&quot;;
text-align: left;</string>
   </property>
   <property name="text">
    <string>OPTIONS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bolusIcon">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>315</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="optionsIcon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>315</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
text-align: left;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="chargingIcon">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;DejaVu Sans&quot;;
color: rgb(87, 227, 137);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>â†‘</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>158</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>158</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>00/00/0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="bolusActivatedLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>325</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;DejaVu Sans&quot;;
color: rgb(98, 160, 234);
</string>
   </property>
   <property name="text">
    <string>BOLUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="bolusActivatedDotsLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>325</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;DejaVu Sans&quot;;
color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string>XXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="bolusActivatedRequestingLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Requesting Label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bolusActivatedStopButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>315</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(192, 28, 40);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QChartView" name="glucoseChartView" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0"/>
  </widget>
  <widget class="QLabel" name="latestReadingLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>mmol/L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="timeWindowButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>3 HRS</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusIconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bolusIndicator">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>iobDecorButton</zorder>
  <zorder>batteryBar</zorder>
  <zorder>batteryLabel</zorder>
  <zorder>insulinBar</zorder>
  <zorder>insulinLevelLabel</zorder>
  <zorder>settingsButton</zorder>
  <zorder>settingsLabel</zorder>
  <zorder>iobLabel</zorder>
  <zorder>iobDectorButton2</zorder>
  <zorder>iobUnitsLabel</zorder>
  <zorder>iobTimeLabel</zorder>
  <zorder>iobUnitsUnit</zorder>
  <zorder>iobUnits2</zorder>
  <zorder>iobUnits</zorder>
  <zorder>iobTime</zorder>
  <zorder>bolusButton</zorder>
  <zorder>optionsButton</zorder>
  <zorder>bolusIcon</zorder>
  <zorder>optionsIcon</zorder>
  <zorder>chargingIcon</zorder>
  <zorder>timeLabel</zorder>
  <zorder>dateLabel</zorder>
  <zorder>bolusActivatedLabel</zorder>
  <zorder>bolusActivatedDotsLabel</zorder>
  <zorder>bolusActivatedRequestingLabel</zorder>
  <zorder>bolusActivatedStopButton</zorder>
  <zorder>glucoseChartView</zorder>
  <zorder>latestReadingLabel</zorder>
  <zorder>label</zorder>
  <zorder>timeWindowButton</zorder>
  <zorder>statusIconLabel</zorder>
  <zorder>bolusIndicator</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
